### PR TITLE
Corrected bug for close accuracy for duplicate letters

### LINKED ISSUE #1: Not indicating the correct number of duplicated letters
ABBEY actual word. if BOBBY is the guess, then it shows yellow, grey, green, yellow, green (pic attached below)
image

it should be yellow, grey, green, gray, green. It shouldn't highlight the 3rd B in BOBBY as yellow.

### PR DESCRIPTION
**Pull request by smathew-23**

### CODE REVIEW

**Review by pdurango — DISMISSED** [review]
What are all these random files?

File: WordleTestProject/UnitTest1.cs [hunk 1]
```diff
@@ -5,17 +5,24 @@ namespace WordleTestProject
...
 
         [Test]
-        public void Test1()
+        public void WordleGuess_Fail()
         {
-            WordleClone wordle = new WordleClone("tower");
-
-            wordle.GuessWord("flees");
 
         }
```
Comment (pdurango) [inline WordleTestProject/UnitTest1.cs]: Please remove or implement

File: WordleCLI/Program.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@ class WorldeCLI
         static void Main(string[] args)
         {
             Console.WriteLine("Welcome to Worlde.");
-            WordleClone wordle = new WordleClone("tower");
+            WordleClone wordle = new WordleClone("abbey");
```
Comment (pdurango) [inline WordleCLI/Program.cs]: Remove the parameter value, the constructor should take in nothing.

**Comment by pdurango** [comment]
LGTM

**Comment by smathew-23** [comment]
> What are all these random 

These were the files you created to show me an example :P anyways, deleted!

**Review by pdurango — APPROVED** [review]
LGTM. P.s. I love you

**Pull request merged**